shipyard/shipyard-cli
# Feat/telepresence support

**Steps to test:**

- Check out this [shipyard](https://github.com/shipyard/shipyard-cli/tree/feat/telepresence-support) cli branch
- export SHIPYARD_BUILD_URL="http://localhost:8080/api/v1"
- turn on telepresence_enabled for your org
- deploy app
- go run main.go get environments
- go run main.go telepresence connect --env 8ee7d148-a0da-4c32-bdc4-bfda4f7e76e6 which will fail
- run telepresence quit
- telepresence connect --kubeconfig ~/.shipyard/kubeconfig
- run telepresence list and you should see all pods in your namespace
